altair/utils/schemapi.py

```diff
@@ -220,20 +220,15 @@ def _prepare_refs(d: Dict[str, Any]) -> Dict[str, Any]:
         for key, value in d.items():
             if key == "$ref":
                 d[key] = _VEGA_LITE_ROOT_URI + d[key]
-            else:
-                # $ref values can only be nested in dictionaries or lists
```

Comment:
Comment block lost

Reply:
See https://github.com/vega/altair/pull/3431#discussion_r1631089964 as that change produces this code.